src/main/webapp/js/server-table.js

```diff
@@ -86,7 +86,7 @@ function refreshServerTable() {
             a.href = server.frontEndUrl;
             a.target = "_blank";
             a.classList.add("gt-external-link");
-            a.appendChild(document.createTextNode(server.name));
+            a.innerHTML = server.name;
             a.appendChild(generateSVGIcon("gerrit-symbol-link"));
```

Comment:
Any security risk for server name?

Reply:
No, It was using innerHTML before, the server.name is already escaped